### DISCUSSION TITLE
Projecting new data onto an existing UMAP

### DISCUSSION THREAD

**Discussion by timoast** [satijalab/seurat#4004] [Q&A] [answered]
How can I project a new dataset onto an existing UMAP?

**Comment by satijalab** [comment, marked as answer]
You can project new data onto an existing UMAP, provided the UMAP model has been stored in the existing UMAP, by running the `MapQuery()` function. You can find an example of how to do this in our [Reference mapping vignette](https://satijalab.org/seurat/articles/integration_mapping.html#unimodal-umap-projection)

Note that the `MapQuery` function can also be used to transfer cell labels from a reference dataset onto a query (i.e. automatically ‘annotate’ a query dataset).
